Gopkg.toml

```diff
@@ -30,10 +30,6 @@
   name = "github.com/hashicorp/go-multierror"
   version = "1.0.0"
 
-[[constraint]]
```

Comment:
does the opencensus lib need to be here?

Reply:
:+1:  added

Reply:
I've been adding `dep ensure -check` to travis builds. Though I guess the build will just fail.